logstash-output-dogstatsd.gemspec

```diff
@@ -11,7 +11,7 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
 
   # Files
-  s.files = `git ls-files`.split($\)+::Dir.glob('vendor/*')
+  s.files = Dir['lib/**/*','spec/**/*','vendor/**/*','*.gemspec','*.md','CONTRIBUTORS','Gemfile','LICENSE','NOTICE.TXT']
```

Comment:
I kept it consistent with other plugins, that all seem to include the two directories besides them not being necessary or even consistent. Will gladly remove it though.

Still testing the changes out, will update the PR later today.